src/Runtime.Base/src/System/Runtime/InteropServices/DllImportAttribute.cs

```diff
@@ -9,6 +9,10 @@ public sealed class DllImportAttribute : Attribute
     {
         public CallingConvention CallingConvention;
 
+        public string EntryPoint;
+
+        public bool SetLastError;
```

Comment:
Will this be needed? It would be ideal if Test.CoreLib doesn't attempt to support any sort of complex marshalling (by complex I mean the kinds that need to call helpers - things like Unicode string marshalling is fine, so are blittable arrays or bool, but SetLastError requires bringing over helpers around the Marshal class)

Reply:
I have an analyzer that says SetLastError MUST be false to be clear to the user. But we can omit if you prefer.

Reply:
I've removed it.